src/index.ts

```diff
@@ -16,4 +16,4 @@ export { default as compile } from "./compile.js";
 export { default as parse } from "./parse.js";
 export { default as render, renderAsync } from "./render.js";
 export { templates } from "./containers.js";
-export { config, config as defaultConfig, getConfig, configure } from "./config.js";
+export { EtaConfig, config, config as defaultConfig, getConfig, configure } from "./config.js";
```

Comment:
Could you change this to `export type { EtaConfig }` on a new line? I think that will make it clearer that this is a type export.

Reply:
Sure! Fixed.